src/Contract/ResponseInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\ReCaptcha\Contract;
+
+use Laminas\Http\Response as HTTPResponse;
+
+interface ResponseInterface
+{
+    /** @return string[] */
+    public function getErrorCodes(): array;
```

Comment:
This interface doesn't seem to provide value: extracting it for the sake of it seems to bring in more BC issues, rather than fixing any